DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md

```diff
@@ -0,0 +1,401 @@
+# `Platform Designer Standard` Sample
+
+This example design shows how to use an FPGA IP produced with the Intel® oneAPI DPC++/C++ Compiler with the Intel® Quartus® Prime Standard Edition to target a Cyclone® V FPGA.
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Ubuntu* 20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10 <br> Windows Server* 2019
+| Hardware                          | This process applies to the [Cyclone® V FPGA and SoC FPGA](https://www.intel.com/content/www/us/en/products/details/fpga/cyclone/v.html)
```

Comment:
mention the specific development kit that you target so that your customer knows which board to use, like the OG platform desginer sample